README.md

```diff
@@ -46,6 +46,17 @@ EOF
 Please refer to [Building the Operator Image](#building-the-operator-image) to
 build your own image.
 
+## Prerequisites
```

Comment:
This header was referenced above ("...Please make
sure the [prerequisites](#Prerequisites) are installed first...")